scripts/tasks.py

```diff
@@ -0,0 +1,46 @@
+import os
+from pathlib import Path
+
+from invoke import task
+from invoke.context import Context
+from rich.console import Console
```

Comment:
We'll probably need to add `rich` into pyproject.toml. You can achieve it through

```shell
poetry add rich --group dev
```

Reply:
Thanks for reminder ! 
This is updated here: [build(pyproject.toml): add rich in dev.dependencies](https://github.com/commitizen-tools/commitizen/pull/1048/commits/f5c6fd7e932cdee5aa60e294c407849c59099ca1)